assets/js/components/KeyMetrics/KeyMetricsSetupCTAWidget.js

```diff
@@ -89,23 +82,13 @@ function KeyMetricsSetupCTAWidget( { Widget, WidgetNull } ) {
 
 	const { navigateTo } = useDispatch( CORE_LOCATION );
 	const openMetricsSelectionPanel = useCallback( () => {
-		if ( isConversionReportingEnabled ) {
-			navigateTo( fullScreenSelectionLink );
-		} else {
-			setValue( KEY_METRICS_SELECTION_PANEL_OPENED_KEY, true );
-		}
+		navigateTo( fullScreenSelectionLink );
```

Comment:
This `navigateTo` should go after the `trackEvent` function call + we need to add `await` to wait until the event is being tracked and before we start navigation.

Reply:
@eugene-manuilov this has been resolved in my latest commits.